mep3_simulation/webots_data/worlds/eurobot_2022.wbt

```diff
@@ -4,8 +4,8 @@ WorldInfo {
   basicTimeStep 16
 }
 Viewpoint {
-  orientation 0.20868380639246203 -0.4933479470449755 -0.8444281331741982 2.455462865957651
-  position -0.970213014458458 1.0796919983826436 1.0058115311780271
+  orientation 0.38422839787751045 -0.16301559922396378 -0.9087323327989015 0.8736019268801365
+  position -1.69821721320748 -1.1607253687432857 1.8928183976533786
```

Comment:
[Ovo](https://github.com/memristor/mep3/pull/13#discussion_r748903651) kaze Darko. Vrati na to sto je bilo ;)

Reply:
Prepravljeno :)